app/src/main/res/values/strings.xml

```diff
@@ -1,5 +1,5 @@
 <resources>
-    <string name="app_name">Open Event Organizer</string>
+    <string name="app_name">eventyay organizer</string>
```

Comment:
creating a separate issue for that. @niranjan94 
Actually, I am not sure how to do it. You mean when building from command line, the app name will be passed as a parameter in the command. right?